superset-frontend/src/dashboard/containers/DashboardPage.tsx

```diff
@@ -42,6 +42,8 @@ const DashboardContainer = React.lazy(
     ),
 );
 
+const documentTitle = document.title;
```

Comment:
Can we rename it to something like `originalDocumentTitle` or `prevDocumentTitle` to make the intention more clear?